NNSharp/IO/ReaderKerasModel.cs

```diff
@@ -203,7 +203,7 @@ private List<IKernelDescriptor> ReadDescriptors(JObject model)
                         descriptor = new TanH();
                         break;
                     case "LeakyReLU":
-                        descriptor = new LeakyReLu(1);
+                        descriptor = new LeakyReLu(0.3);
```

Comment:
itt egy változónak kellene lennie, nem pedig egy konstansnak. A LeakyReLu-nak tetszőleges paramétere is lehet Kerasban. Nézd meg, hogy van implementálva egy layer a C# oldalon. Ez nem egy szokványos aktiváció ilyen értelemben.